scripts/tfw_lib.sh

```diff
@@ -151,8 +151,9 @@ tfw_set_net_queues()
 	cpu_mask=$(perl -le 'printf("%x", (1 << '$CPUS_N') - 1)')
 
 	for dev in $devs; do
-		queues=$(ethtool -l $dev 2>/dev/null \
+		queues_str=$(ethtool -l $dev 2>/dev/null \
 				| grep -m 1 RX | sed -e 's/RX\:\s*//')
+		queues=$(printf '%d' "$queues_str")
```

Comment:
We still have error from `printf()` this time
```
./scripts/tfw_lib.sh: line 156: printf: n/a: invalid number
```
You can check the script even manually - just run the commands by hands